src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
Just changing from abstract class to trait is a BC break for dev extending this concerning class.